admin/create-theme/theme-styles.php

```diff
@@ -17,22 +17,36 @@ public static function update_style_css( $style_css, $theme ) {
 		$author        = stripslashes( $theme['author'] );
 		$author_uri    = $theme['author_uri'];
 		$wp_version    = get_bloginfo( 'version' );
+		$wp_min        = $current_theme->get( 'RequiresWP' );
 		$version       = $theme['version'];
 		$requires_php  = $current_theme->get( 'RequiresPHP' );
 		$template      = $current_theme->get( 'Template' );
 		$text_domain   = $theme['slug'];
+		$license       = 'GNU General Public License v2 or later';
+		$license_uri   = 'http://www.gnu.org/licenses/gpl-2.0.html';
+		$tags          = CBT_Theme_Tags::theme_tags_list( $theme );
+		$copyright     = '';
+
+		preg_match( '/License: (.+)/', $style_css, $matches );
+		if ( isset( $matches[1] ) ) {
+			$license = $matches[1];
+		}
+		preg_match( '/License URI: (.+)/', $style_css, $matches );
+		if ( isset( $matches[1] ) ) {
+			$license_uri = $matches[1];
+		}
```

Comment:
Wouldn't it be more accurate and simpler to use `get_file_data()`? The `WP_Theme` class also seems to use this function [internally](https://github.com/WordPress/wordpress-develop/blob/13d5244c43d65da5fd3ff7abd390fa36bb4a87b5/src/wp-includes/class-wp-theme.php#L348).

```php
$theme_data = get_file_data(
	get_stylesheet_directory() . '/style.css',
	array(
		'License' => 'License',
		'License URI' => 'License URI',
	)
);
```

Reply:
Nice.  I didn't even know about get_file_data, so yeah.  That does look simpler. :)